pkg/analyse/grafana.go

```diff
@@ -114,14 +115,55 @@ func metricsFromTemplating(templating sdk.Templating, metrics map[string]struct{
 	return parseErrors
 }
 
+// Workaround to support Grafana "timeseries" panel. This should
+// be implemented in grafana/tools-sdk, and removed from here.
+func getCustomPanelTargets(panel sdk.Panel) *[]sdk.Target {
+	if panel.CommonPanel.Type != "timeseries" {
+		return nil
+	}
+
+	// Heavy handed approach to re-marshal the panel and parse it again
+	// so that we can extract the 'targets' field in the right format.
+
+	bytes, err := json.Marshal(panel.CustomPanel)
+	if err != nil {
+		log.Debugln("msg", "panel re-marshalling error", "err", err)
+		return nil
+	}
+
+	type panelType struct {
+		Targets []sdk.Target `json:"targets,omitempty"`
+	}
+
+	var parsedPanel panelType
+	err = json.Unmarshal(bytes, &parsedPanel)
+	if err != nil {
+		log.Debugln("msg", "panel parsing error", "err", err)
+		return nil
+	}
+
+	return &parsedPanel.Targets
+}
+
+func getPanelTargets(panel sdk.Panel) *[]sdk.Target {
```

Comment:
Nit, but why not just inline this function considering it's basically an `if` condition?